articles/encryption-in-mongodb/index.md

```diff
@@ -0,0 +1,145 @@
+
+For more information, you can visit MongoDB official documentation here <https://docs.mongodb.com/manual/tutorial/configure-ssl/>
+
+### Encryption Performance in MongoDB
+
+When choosing to encrypt your MongoDB database, users should consider performance. Performance is a very important factor especially for developers who store large amounts of data that customers access daily through front-end applications. When a banking or retail application requests thousands or millions of records from a database daily, any latency or downtime can seriously impact business continuity and operations.
+
+As for today's scenario, we want fast retrieval of data and if an application is not meeting this need is not at all accepted.
+This is why MongoDB has conducted performance tests using Intel Xeon X5675 CPUs thus storage engine experiences an average latency between 10%-15%, depending on the amount of data that a user is reading or writing to the database.
+
+When the user writes only large amounts of data to the database, the performance impacts fall on the server-side, however, in more commons scenarios a user might be performing read-only commands for the data so the performance will likely fall between 5%-10%.
+
+### Encryption Key Management in MongoDB
+
+Encryption key management is the method used to protect and manage your encryption keys. Enterprise encryption key management should meet the key management framework and recommendations as suggested by NIST in Special Publications.
+MongoDB does not include an enterprise encryption key management solution, and users must purchase a solution from a third-party key management solution provider. MongoDB Enterprise Editions and enables customers to protect encryption using several tested and validated enterprise key management partners which is a very plus point of MongoDB.
+
+Now you may ask why key management is necessary so here is the answer,
+
+Encryption key management is necessary because without key management you left the keys to protecting your sensitive business and customer data exposed, then you expose your entire organization to the risk of data loss or theft.
+Today, with MongoDB Enterprise, customers can meet encryption and key management best practices easily through implementing native encryption and deploying a third-party enterprise key management solution.
+
+Before we moving on to how to implement SSL encryption in your database let's first understand what is SSL.
+
+### What is SSL?
+
+SSL stands for Secure Sockets Layer, and it refers to a protocol for encrypting and securing communications that take place on the Internet.
+The purpose of SSL is to secure communications between a client and a server, but it can also secure email, VoIP, LTE, and other communications on a shaky and untrusted network.
+
+Now I will tell you how to add SSL transport encryption to your MongoDB database.
```

Comment:
I prefer you use show instead of tell. You can also choose to rephrase it to, _Now we will learn how to add SSL..._